e2e/terraform/packer/build

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+set -e
+
+usage() {
+    cat <<EOF
+Usage: build <target>
+
+Build an AMI for the target configuration
+
+Examples
+  build ubuntu-bionic-amd64
+
+EOF
+
+    exit 2
+}
+
+if [[ $# -ne 1 ]]; then
+     usage
+fi
+
+target=${1/%.pkr.hcl/}
+
+directory=$(dirname "$0")
+cd ${directory}
+
+if ! test -f "${target}.pkr.hcl"; then
+    echo "${target}.pkr.hcl is not present" >&2
+    exit 1
+fi
+
+sha=$(git log -n 1 --pretty=format:%H ${dirname})
```

Comment:
I opted to use the latest sha of the `packer` directory.  I could have used fine-grain shas for each image type to avoid rebuilding them.  However, I thought the overhead will be significant, once we start adding more image types (e.g. multiple OSes/archs/etc), and having a single reference SHA for all images seem fine given that we don't change the packer scripts much. We can tweak later if it's a problem.